Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -500,7 +502,9 @@ import BraintreeDataCollector
         var urlComponents = URLComponents(url: payPalAppRedirectURL, resolvingAgainstBaseURL: true)
         let additionalQueryItems: [URLQueryItem] = [
             URLQueryItem(name: "source", value: "braintree_sdk"),
-            URLQueryItem(name: "switch_initiated_time", value: String(Int(round(Date().timeIntervalSince1970 * 1000))))
+            URLQueryItem(name: "switch_initiated_time", value: String(Int(round(Date().timeIntervalSince1970 * 1000)))),
+            URLQueryItem(name: "flow_type", value: isVaultRequest ? "va" : "ecs"),
+            URLQueryItem(name: "merchant", value: merchantAccountId ?? "unknown")
```

Comment:
nit: can probably remove the "unknown" default value here
```suggestion
            URLQueryItem(name: "merchant", value: merchantAccountId)
```

Reply:
also looks like we're not sending in an "unknown" in the android pr 👀

Reply:
That's right: in Android `merchantID` is non optional, so they don't need to provide placeholder value.
In case of iOS when `merchantAccountId` is nil, merchant query item will result in 
`.../app-switch-checkout?/token=ABCD&merchant`

Vs after providing default value:
.../app-switch-checkout?/token=ABCD&merchant=unknown

Reply:
Have we confirmed these values are coming through as expected?